fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubber.java

```diff
@@ -263,8 +287,32 @@ public Builder useLegacyScrubber(boolean legacy) {
                 return this;
             }
 
+            /**
+             * Choose a specific "Already Scrubbed" rangeSet prefix. This may be useful for
+             * multiple scrubbing jobs that may not affect each other.
```

Comment:
I feel like referring to the range set is getting more into the implementation details than we want, perhaps this would be clearer:
```suggestion
             * Choose a specific id for this scrubbing operation. If the scrubbing stops, or fails, it can be resumed at a later time by constructing the same builder. Work done with other ids will not affect work done with this id.
```

Reply:
:whi